bradley-childs.md

```diff
@@ -0,0 +1,45 @@
+
+## Sudha R Ponnaganti
+
+I have worked with Brad for a short period time during one of OpenShift Releases. He is a true champion and always responded patiently if we need something to close on outstanding issues. I always relied on his quick responses and confident that he would get things done and always call up on him if we needed a fix for an issue that needs to be done quickly. It is a loss for his work family. My deepest condolences to the family and friends of Brad.
+
+## Hemant Kumar
+
+Brad was someone who believed in people. He believed in a person's ability to grow. He was fun, easy going and somehow untouched by ego. He was not only my team leader but a good friend. I am going to miss him immensely.
+
+## Jose A. Rivera
+
+Brad helped me through a bit of a rough patch in my career, and for that I'll be forever grateful. I was always looking forward to the next time I could hang out with him. He also made a mean batch of chili. I'll sure as heck miss both.
+
+## Mike Barrett
+
+Brad helped me more than I helped him.  Which is a sign of a good person.  He helped give the open source world the ability to save something and to have state; to stay on disk.  He could see his way through a technical issue better than most.  His wit was sharp and his humor...man, he made me laugh.  I will miss his laugh in this world.  You will definitely stay in my memory.  Thanks for all that you did for the world.
+
+## Joe Beda
+
+I didn’t have the opportunity to work with Brad. It is clear, however, that he built and sustained community by helping others.  Careerwise, I’m late to open source. It is people like Brad, however, that make it a welcoming place and show that this is all more than just a job. I wish I had a chance to get to know him.
+
+## Erin Boyd
+
+Brad and I knew each other briefly at IBM, and then became friends as we worked closely together at Red Hat. We often disagreed about design and implementations, but our mutual respect for one another allowed us to work through high stress situations to find the best solution. Outside of work we talked about life and I was often trying to set Brad up with friends of mine or random people I met at conferences. We joked I was his ‘wing woman.’ He was a sounding board for me, as I was for him when we would become frustrated with grind. The last time I saw Brad we were sitting outside on the patio at Barcelona telling me I needed to relax more and live in the moment. He would almost always see through the bs and call people out on it and I will miss his genuine spirit.
```

Comment:
## Saad Ali

Brad and I served as co-chairs of Kubernetes SIG Storage. We worked together to shepard many of the great projects the SIG delivered including the PV/PVC controller rewrite, creation of the attach/detach controller, dynamic volume provisioning with StorageClasses, and so much more. Brad was always so easy to work with, and willing to help. Beyond being a dependable lead for SIG Storage, I remember Brad as being quick with a joke and just a fun person to be around. He knew how to lighten the mood and defuse tension when there were big disagreements in the group. I always looked forward to talking to seeing him at conferences. You will be missed Brad, rest in peace!